app/js/components/collection_widget.ts

```diff
@@ -125,6 +126,20 @@ class CollectionWidget {
     $addEntryButton.on('click', handleAddClick);
   }
 
+  /**
+   * Add click handler to add entry
```

Comment:
Technically this is not a click handler

Reply:
Changed the name to describe it's usage better.